CONTRIBUTING.md

```diff
@@ -75,6 +109,55 @@ $ go test -v -short ./...
 
 ## Releasing
 
+### Notes on versioning
+
+We version the suite using the syntax `1.x.y+suite.z`, where `1.x.y` is the version of
+the Conjur server included in the suite.
+
+This works as follows:
+
+- When there is sufficient content in the Conjur OSS suite that stakeholders
+  determine a suite release is merited, a new suite release is prepared with version
+  `1.x.y+suite.z`, where `1.x.y` is the version of [Conjur](https://github.com/cyberark/conjur)
+  included in the release.
+  - Note: suite releases will not necessarily happen for every Conjur core release.
+
+- If this is the first suite release for Conjur version `1.x.y`, then the suite
+  release will be versioned `1.x.y+suite.1`, where `1.x.y` matches the included
+  Conjur core version.
+
+- At the next point in time when there is sufficient content in the Conjur OSS
+  suite that stakeholders determine a suite release is merited:
+  - If the Conjur core version in the suite is still `1.x.y`, then the next suite
+    release version will be `1.x.y+suite.2`.
+  - If the Conjur core version has changed to `1.n.m`, then the next suite
+    release version will be `1.n.m+suite.1`.
+
+Additional notes:
+- If Conjur changes its version with a new **minor or patch** release, we _may_
+  have a new suite release, but it is not required unless the stakeholders agree.
+- If Conjur changes its version with a new **major** release, we _must_ have a
+  new suite release, and stakeholders will determine how long we will continue
+  to support old version of Conjur in the suite.
+- In any Conjur suite release, if the Conjur core version is the same as the last
+  suite release, the `.z` digit in the suite version will be incremented.
```

Comment:
Isn't this restating what you have above?  Do you think this extra clarification is needed?

Reply:
it might not be, but it doesn't hurt?